tests/Resources/ti.addontest.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'); // eslint-disable-line no-unused-vars
+
+describe('Error', function () {
+	it.windowsMissing('JavaScript exceptions', function () {
+		var e = {};
+
+		try {
+			Ti.API.info(e.test.crash);
+			should.fail('Expected to throw JavaScript exception');
+		} catch (ex) {
+			// has "message" property
+			ex.should.have.readOnlyProperty('message').which.is.a.String;
+			// has "stack" property
+			ex.should.have.readOnlyProperty('stack').which.is.a.String;
+            // does not have native stack trace
+			ex.should.not.have.property('nativeStack');
+		}
+	});
+
+	it.windowsMissing('Native exceptions', function () {
+		try {
+			Ti.Geolocation.accuracy = null;
+			should.fail('Expected to throw native exception');
+		} catch (ex) {
+			// has "message" property
+			ex.should.have.readOnlyProperty('message').which.is.a.String;
+			// has "stack" property
+            ex.should.have.readOnlyProperty('stack').which.is.a.String;
+            // has special "nativeStack" property for native stacktrace
+            // TODO: Should we make this an array instead? 
+			ex.should.have.property('nativeStack').which.is.a.String;
```

Comment:
I would make this array to make it easier to format to string on crash dialog. On Windows I am implementing it that way.

Reply:
I think having it look/feel similar to the standard `stack` property makes sense though too.
I understand that an array would be easier to loop through and would avoid EOL differences, but originally I think the goal was to have a native equivalent of the js stack property which is a string (and on V8/Android actually has the type of Error and message prefixed ahead of the actual stacktrace)

Reply:
The question really is how developers use it in production, and I think a common use-case is logging it to the console (if even used), so a String might be more convenient as well. But I understand @infosia that it feels "wrong" from a structural perspective.

Reply:
ok makes sense, considering to align with `stack` property, I'm ok with string.